index.html

```diff
@@ -3152,11 +3178,15 @@ <h5>Existing implementations</h5>
 </li>
 <li>Mouse wheel movements (usually used to scroll the page) are captured
 and converted into continuous zoom adjustments
-(mouse scroll down becomes zoom in, mouse scroll up becomes zoom out).
+(mouse scroll down becomes zoom out, mouse scroll up becomes zoom in).
+Apple Mapkit JS inverts this popular convention.
+Modifier keys (such as Ctrl for Google Maps and Shift for Apple Mapkit JS)
+may be required so as not to intefere with default scroll behavior.
 </li>
 <li>Touch gestures for pinch zoom are used to zoom the map instead of the page.
 </li>
 </ul>
+<p>It should be </p>
```

Comment:
Is this trying to convey an uncertainty of the supported tools or is something missing in the sentencing?

Reply:
Good catch,  just a stray sentence fragment that I've removed!